xbmc/cores/VideoPlayer/Process/VideoBuffer.cpp

```diff
@@ -318,14 +318,21 @@ void CVideoBufferPoolSysMem::Return(int id)
   m_free.push_back(id);
 }
 
-void CVideoBufferPoolSysMem::Configure(AVPixelFormat format, int width, int height)
+void CVideoBufferPoolSysMem::Configure(AVPixelFormat format, int width, int height, int aligned_width, int aligned_height, int size)
```

Comment:
Ah - iLineSize/data has gone from VideoPicture, so can't use that for renderer to work out what the aligned width and height are.
So I need the CVideoBuffer (or mmal derived class) to contain this information and some way for AddonVideoCodec to pass that in. Configuring the pool (which the CVideoBuffer has access to) seems like the obvious way.